apps/emqx_modules/src/emqx_delayed.erl

```diff
@@ -147,6 +149,9 @@ enable() ->
 disable() ->
     gen_server:call(?SERVER, disable).
 
+is_enabled() ->
+    gen_server:call(?SERVER, is_enabled).
```

Comment:
this call may fail with `timeout` exception when process is busy.
or `noproc` exception if telemetry is started earlier than `emqx_delayed`

Reply:
You're right.  And retainer has the same problem.  I think I'll change both to use an application env flag value to indicate that.  Thanks for pointing it out!  :beers: